game/Utils.ts

```diff
@@ -225,6 +225,13 @@ export class Utils {
         return [spawnInputs, spawnProof];
     }
 
+    /*
+     * Wrapper for genRandomSalt();
+     */
+    static genRandomInt(): bigint {
```

Comment:
nit: potentially chance genRandomSalt to return a bigint instead? do we ever need it otherwise? 

mainly bc genRandomInt doesn't have the range implied in the function title. (vs salt you know it's either bn128 scalar field or uint256)